maps/glloydstation/fluff/species_overrides.dm

```diff
@@ -29,33 +29,52 @@
 	)
 
 //machines
+/singleton/cultural_info/culture/ipc
+	description = "Positronic intelligence is an emerging technology in the workforce, and you are one of the prototypes. \
+	IPCs (Integrated Positronic Chassis) are a loose category of self-willed robots with a humanoid form. They are \
+	reliable and dedicated workers, albeit more than slightly inhuman in outlook and perspective. <br><br> Typical positronic brains \
+	are roughly as intelligent as a human on the smarter side of average, and are fast learners.
+
 /datum/species/machine
 	available_cultural_info = list(
 		TAG_CULTURE = list(
 			CULTURE_POSITRONICS
 		),
 		TAG_HOMEWORLD = list(
-			HOME_SYSTEM_ROOT,
-			HOME_SYSTEM_LUNA,
 			HOME_SYSTEM_MARS,
+			HOME_SYSTEM_LUNA,
 			HOME_SYSTEM_VENUS,
 			HOME_SYSTEM_CERES,
 			HOME_SYSTEM_PLUTO,
 			HOME_SYSTEM_TAU_CETI,
+			HOME_SYSTEM_HELIOS,
+			HOME_SYSTEM_TERRA,
+			HOME_SYSTEM_TERSTEN,
+			HOME_SYSTEM_LORRIMAN,
+			HOME_SYSTEM_CINU,
+			HOME_SYSTEM_YUKLID,
+			HOME_SYSTEM_LORDANIA,
+			HOME_SYSTEM_KINGSTON,
+			HOME_SYSTEM_GAIA,
+			HOME_SYSTEM_RYCLIES,
+			HOME_SYSTEM_READE,
+			HOME_SYSTEM_PROCYON,
 			HOME_SYSTEM_OTHER
 		),
 		TAG_FACTION = list(
-			FACTION_POSITRONICS,
-			FACTION_SOL_CENTRAL,
+			FACTION_OTHER,
 			FACTION_NANOTRASEN,
-			FACTION_FREETRADE,
 			FACTION_XYNERGY,
-			FACTION_OTHER
+			FACTION_HEPHAESTUS,
+			FACTION_SOL_CENTRAL,
+			FACTION_INDIE_CONFED,
```

Comment:
do we want IPCs from UHA?

Reply:
I realize I wrote this in the Glloydstation file, even though this applies to Nerva, but this raises another question. Will this cause an issue, seeing as FACTION_INDIE_CONFED isn't available in glloydstation (see maps\glloydstation\fluff\_fluff.dm)?

Reply:
knew I forgot something. yeah faction uha is a bit too far and i copypasted too close to the sun in glloydstation overrides